applications/sandbox/systems/testsystem.cpp

```diff
@@ -122,7 +125,6 @@ void TestSystem::selectAdapter()
         {
             log::info("Adapter selected");
             m_adapter = adapter;
-            break;
```

Comment:
so we take the last discrete gpu now?

Reply:
I suppose yeah, not really intentional but I needed it to list every adapter when I was testing :p

The order of the adapters is unspecified so it doesnt really matter which we pick here, I'll update the sandbox code with some better selection code from the adapter selection sample later